stubs/redis/redis/commands/core.pyi

```diff
@@ -8,13 +9,17 @@ from ..client import _Key, _Value
 _ScoreCastFuncReturn = TypeVar("_ScoreCastFuncReturn")
 _StrType = TypeVar("_StrType", bound=Union[str, bytes])
 
-class CoreCommands(Generic[_StrType]):
-    def acl_cat(self, category: str | None = ...) -> list[str]: ...
-    def acl_deluser(self, username: str) -> int: ...
-    def acl_genpass(self, bits: int | None = ...) -> str: ...
-    def acl_getuser(self, username: str) -> Any | None: ...
-    def acl_list(self) -> list[str]: ...
-    def acl_load(self) -> bool: ...
+class ACLCommands(Generic[_StrType]):
+    def acl_cat(self, category: str | None = ..., **kwargs) -> list[str]: ...
+    def acl_deluser(self, *username: str, **kwargs) -> int: ...
+    def acl_genpass(self, bits: int | None = ..., **kwargs) -> str: ...
```

Comment:
Should this return `str` or `_StrType`? This also applies to `acl_whoami()`. How about the methods that return a list?

Reply:
It's quite possible that `_StrType` is correct here. I moved these definitions over from `CoreCommands` and didn't investigate further. I'd like to punt the investigation to a potential followup PR, though.